src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -249,6 +260,34 @@ private static void Run(string[] args)
         }
     }
 
+    private static string CheckForDuplicateArgs(string[] args)
+    {
+        Dictionary<string, string> argPairs = new Dictionary<string, string>();
+        for (int i = 0; i < args.Length; i++)
+        {
+            string arg = args[i];
+
+            if (arg.StartsWith("-"))
+            {
+                string key = arg.TrimStart('-');
+
+                if (i + 1 < args.Length && !args[i + 1].StartsWith("-"))
+                {
+                    string value = args[i + 1];
+                    if (argPairs.ContainsKey(key)){
+                        return key;
+                    }
+                    argPairs[key] = value;
+                    i++;
+                }
+                else
+                {
+                    argPairs[key] = string.Empty;
+                }
+            }
+        }
+        return string.Empty;
+    }
```

Comment:
I think this could be rewritten simpler with a single `GroupBy.Having`.
Also would be good to use same parsing as in `CommandLineApplication`: 
```
          if (b.StartsWith("--"))
            strArray = b.Substring(2).Split(new char[2]
            {
              ':',
              '='
            }, 2);
          else if (b.StartsWith("-"))
            shortOption = b.Substring(1).Split(new char[2]
            {
              ':',
              '='
            }, 2);
```

Reply:
I tried using GroupBy. But that implementation seemed a bit more complicated.

Reply:
To reduce the complexity, I have implemented it some other way.